JS/inject.js

```diff
@@ -1,147 +1,150 @@
 class MessageData {
     constructor(fullContent = "a,b,c") { //signal, service, content
```

Comment:
Question not related to pull request. Does "a,b,c" here represent the default values? If so, are their more appropriate values?